### PR TITLE
fix service docs' jump and format error

### PR DESCRIPTION
**Pull request by xieyanker**
fix service docs' jump and format error

Convert as follows:

before:
image

after:
image

### CODE REVIEW

**Comment by xieyanker** [comment]
https://deploy-preview-19741--kubernetes-io-master-staging.netlify.com/zh/docs/concepts/services-networking/service/

/assign @tengqm

File: content/zh/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ also named “my-service”.
 
 上述配置创建一个名称为 "my-service" 的 `Service` 对象，它会将请求代理到使用 TCP 端口 9376，并且具有标签 `"app=MyApp"` 的 `Pod` 上。
 Kubernetes 为该服务分配一个 IP 地址（有时称为 "集群IP" ），该 IP 地址由服务代理使用。
-(请参见下面的 [虚拟 IP 和服务代理](#virtual-ips-and-service-proxies)).
```
Comment (tanjunchen) [inline content/zh/docs/concepts/services-networking/service.md]: Thanks your contribution.
Please confirm whether other files in zh/ quote  #virtual-ips-and-service-proxies
Or you can add { #virtual-ips-and-service-proxies} directly after this title.
like this https://github.com/kubernetes/website/pull/19741/files#diff-3e859b154c11574e4d22e21b65e12a9fL1062

Comment (xieyanker) [reply content/zh/docs/concepts/services-networking/service.md]: Thank you for your reminder, there are indeed other files using `#virtual-ips-and-service-proxies`, so I add `{#virtual-ips-and-service-proxies}` after thetitle.

Comment (tanjunchen) [reply content/zh/docs/concepts/services-networking/service.md]: sure thanks

**Pull request merged**